Tasks/VsTest/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "vsts-tasks-vstest",
+  "version": "1.0.0",
+  "description": "VSTS VSTEST Task",
+  "main": "VSTest.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/vsts-tasks.git"
+  },
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/vsts-tasks/issues"
+  },
+  "homepage": "https://github.com/Microsoft/vsts-tasks#readme",
+  "dependencies": {
+    "regedit": "2.2.6",	
+    "xml2js": "0.4.16"
```

Comment:
Since this is a new task, use the new 0.8 task lib.  See how the ShellScript task added it here.  We're opting in a task at a time because it had reaction in code for the better api.  Might as well start with it.  Without it, it falls back to using 0.7.2 which had issues around exit.